evap/results/templates/results_evaluation_detail.html

```diff
@@ -125,7 +125,7 @@ <h3>{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
         </div>
     {% endif %}
 
-    {% if contributor_contribution_results %}
+    {% if contributor_contribution_results or other_contributors %}
```

Comment:
the condition below says `view == 'export' and other_contributors `, shouldn't both parts be in here? (An empty card body might be rendered otherwise?)

Reply:
I agree that the condition here and below should be identical, I'd say we should drop the view == 'export' below though. In the view, the `other_contributors` is filled only when view == 'export'

Reply:
Speaking of `other_contributors`: What do you think about renaming the variable? Re-reading the code, it took me quite a while to understand how "other contributors" are different from "non other contributors"^^

I'd maybe `suggest contributors_without_displayed_results` or something similar

Reply:
i removed the `view == 'export'` below.
naming the variable is hard, though. it only makes sense when looking at it from the export view perspective which is always for a dedicated contributor and all other contributors are then "other contributors". "contributors_without_displayed_results" is also not optimal, because there might be a number of contributors without displayed results (because they might not have any or no results are published at all) independent from the export view stuff.

Reply:
I see your point. How about `contributors_with_omitted_results` or variations with `collapsed`, `left_out`, or `skipped`?

If these also have issues, I'm also fine with leaving it as is.

Reply:
`contributors_with_omitted_results` is fine